third_party/rust/indexmap/v1/BUILD.gn

```diff
@@ -14,7 +14,6 @@ cargo_crate("lib") {
 
   # Only for usage from third-party crates. Add the crate to
   # third_party.toml to use it from first-party code.
-  visibility = [ "//third_party/rust/*" ]
```

Comment:
This should be
```gn
visibility = [ "//brave/third_party/rust/*" ]
```
list all the sibling BUILD.gn, rather than left public.

Reply:
Fixed in https://github.com/brave/brave-core/pull/21524/commits/1f2b283553031b8db4f0b673243f7b2a79015a25.